web/src/hooks/useApps.ts

```diff
@@ -1,10 +1,10 @@
-import { AppModel } from "@/lib/models";
```

Comment:
@igorosip0v worth taking a look when you get a chance, I'm out of the loop with frontend